### DISCUSSION TITLE
Global search

### DISCUSSION THREAD

**Discussion by knutrichard** [snipe/snipe-it#13477] [Ideas]
It would have been a great feature if I can search from one global searchfield on anything, anywhere in the database. Example, search for a model number without knowing if it's an accessory, asset, lisence or consumables, and so on.

As it appears today, I can't search for a spesific keyword without knowing what kind it is, enter that category and then search.

**Comment by snipe** [comment]
It’s just not really possible without getting into some really nasty queries, as we’d have to be searching across hundreds of fields across dozens of tables and it would just make things go way too slow.
